modules/zookeeper/src/test/java/org/apache/ignite/spi/discovery/zk/internal/ZookeeperDiscoveryTopologyChangeAndReconnectTest.java

```diff
@@ -562,32 +564,23 @@ public void testForcibleClientFail() throws Exception {
      * @throws Exception If failed.
      */
     @Test
+    @WithSystemProperty(key = IGNITE_SQL_DISABLE_SYSTEM_VIEWS, value = "true")
```

Comment:
`IgniteH2Indexing` connects to H2 instance by URL based on nodeId, so there can be more errors in indexing here in the future even if system views are disabled.
Perhaps it's better to disable indexing at all for this test (something like `GridQueryProcessor.idxCls = DummyQueryIndexing.class;` before each grid start).

Reply:
@alex-plekhanov, thanks for the suggestion. I have fixed the test. Could you take a look? [TC test suite](https://ci.ignite.apache.org/viewLog.html?buildId=4713749&buildTypeId=IgniteTests24Java8_ZooKeeperDiscovery1) looks fine.